src/common.rs

```diff
@@ -20,14 +22,46 @@ pub static WORK_DIR: &str = "/tmp";
 // Secure mount of tpmfs (False is generally used for development environments)
 pub static MOUNT_SECURE: bool = true;
 
+/*
+ * Return: Returns the configuration file provided in the environment variable
+ * KEYLIME_CONFIG or defaults to /etc/keylime.conf
+ *
+ * Example call:
+ * let config = config_file_get();
+ */
+pub(crate) fn config_file_get() -> String {
+    match env::var("KEYLIME_CONFIG") {
+        Ok(cfg) => {
+            // The variable length must be larger than 0 to accept
+            if !cfg.is_empty() {
+                cfg
+            } else {
+                String::from(DEFAULT_CONFIG)
+            }
+        }
+        _ => String::from(DEFAULT_CONFIG),
+    }
+}
+
+/*
+ * Input: [section] and key
+ * Return: Returns the matched key
+ *
+ * Example call:
+ * let port = common::config_get("general","cloudagent_port");
+ */
+pub(crate) fn config_get(section: &str, key: &str) -> Result<String> {
+    explicit_config_get(config_file_get().as_str(), section, key)
+}
+
 /*
  * Input: config file location (e.g. /etc/keylime.conf), [section] and key
  * Return: Returns the matched key
  *
  * Example call:
- * let port = common::config_get("/etc/keylime.conf""general","cloudagent_port");
+ * let port = common::config_get("/etc/keylime.conf", "general","cloudagent_port");
  */
-pub(crate) fn config_get(
+pub(crate) fn explicit_config_get(
```

Comment:
What if this function dropped its `conf_name` parameter and called `config_file_get()`? Then maybe we wouldn't need `config_get()` to wrap this one (and we could change the name back to `config_get()`)?

Reply:
I think that would be totally valid. My thought was originally to keep the older function around in case there was a need but if there isn't any expectation for future need I'm 100% on board with this!

Reply:
Will update with this pattern.